tests/unit/resources/server_status_out.txt

```diff
@@ -9,9 +9,10 @@ Server Status:
 	Node URI(http): http://inactive/stmt
 	Presto Version: presto-main:0.99-SNAPSHOT
 	Node is active: False
+	Connectors:     hive, system, tpch
 Server Status:
 	Node3(IP: IP3 roles: worker): Running
-	No information available
+	No information available: the coordinator has not yet discovered this node
 Server Status:
-	Node4(IP:  roles: worker): Not Running
-	No information available
+	Node4(IP: Unknown roles: worker): Not Running
```

Comment:
While you're here, could you put a comma before roles in the status output, and maybe capitalize roles too?